index.html

```diff
@@ -230,52 +230,79 @@ <h4 class="work-item-title">Multi-Post Stories</h4>
       </section>
       <section class="contact">
         <p class="contact-title desktop">Let's chat!</p>
-        <p class="contact-title mobile">CONTACT US</p>
+        <p class="contact-title mobile">CONTACT ME</p>
         <p class="contact-description">
           If you have an application you are interested in developing, a feature
           that you need built or a project that needs coding. I’d love to help
           with it
         </p>
-        <form class="contact-form">
+        <form
+          class="contact-form"
+          action="https://formspree.io/f/xvolvllr"
+          method="POST"
+        >
           <input
             type="text"
             id="fullname"
             name="fullname"
             placeholder="Enter Full Name"
+            maxlength="30"
           />
           <div id="names">
             <input
               type="text"
               id="fname"
               name="firstname"
               placeholder="Enter First Name"
+              maxlength="30"
+              required
             />
             <input
               type="text"
               id="lname"
               name="lastname"
               placeholder="Last Name"
+              maxlength="30"
+              required
             />
           </div>
           <input
-            type="text"
+            type="email"
             id="email"
             name="Email"
             placeholder="Email Address"
+            required
           />
-          <textarea rows="5" cols="50" name="description">
-          Write your message here
-      </textarea
+          <!-- <textarea
+            placeholder="Write your message here..."
+            rows="5"
+            cols="50"
+            name="description"
+            maxlength="500"
+            required
           >
```

Comment:
- [ ] [optional] I would suggest removing these comments from the code to make your code clean.